### DISCUSSION TITLE
Add a webhook topic for function runs

### DISCUSSION THREAD

**Discussion by mironov** [Shopify/function-examples#391] [Feedback: General Improvements]
It can be helpful to:

- Monitor and fix any errors that occur during runs.
- Debug reported issues such as Cart Transform not expanding a bundle.
- Track usage and provide customers with reports on their bundle sales.

It can be authorized by a new scope or, e.g., by existing `read_orders`.

**Comment by thosakwe** [comment]
This would be a major improvement for providing merchants with analytics.

Currently, if you want to count the number of orders a discount function has processed, there's not a clear solution.

**Comment by sebastianpisula** [comment]
+1
